src/app/pages/openstax-tutor/how-it-works.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+
+export default function HowItWorks({model}) {
+    const blurbs = [1, 2, 3, 4].map((num) => (
```

Comment:
lodash has a handy `range` method for stuff like this https://lodash.com/docs/4.17.15#range